src/builder.rs

```diff
@@ -1065,7 +1065,7 @@ impl<V> Bundle<InProgress<Proven, PartiallyAuthorized>, V> {
 }
 
 #[cfg(any(test, feature = "test-dependencies"))]
-pub mod testing {
+pub(crate) mod testing {
```

Comment:
The intent of the `test-dependencies` feature is that by enabling it, the `testing` module is made available to downstream crates. So I don't think this is the right change.

Reply:
There is nothing public in this module. In fact, there's an `#[allow(dead_code)]` on the strategy in this module. If that is made public at some point, then this can be reverted, but as-is this is an empty module.

Reply:
I'd argue that maybe it's better to make `arb_bundle` public, and remove the `allow`, but I guess this is fine.